cli/tools/repl/channel.rs

```diff
@@ -46,11 +61,41 @@ impl RustylineSyncMessageSender {
     params: Option<Value>,
   ) -> Result<Value, AnyError> {
     if let Err(err) =
-      self.message_tx.blocking_send((method.to_string(), params))
+      self
+        .message_tx
+        .blocking_send(RustylineSyncMessage::PostMessage {
+          method: method.to_string(),
+          params,
+        })
     {
       Err(anyhow!("{}", err))
     } else {
-      self.response_rx.borrow_mut().blocking_recv().unwrap()
+      match self.response_rx.borrow_mut().blocking_recv().unwrap() {
+        RustylineSyncResponse::PostMessage(result) => result,
+        _ => unreachable!(),
```

Comment:
```suggestion
        RustylineSyncResponse::LspCompletions(_) => unreachable!(),
```
To ensure the compiler croaks when a new variant is added to the enum.